earthpy/spatial.py

```diff
@@ -432,7 +450,8 @@ def plot_rgb(arr, rgb=(0, 1, 2),
         arr_rescaled = np.zeros_like(rgb_bands)
         for ii, band in enumerate(rgb_bands):
             p2, p98 = np.percentile(band, (s_min, s_max))
```

Comment:
The names `p2` and `p98` only make sense for the default value of `str_clip`. Why not call these `low` and `high` or `lower_lim` and `upper_lim`?

Reply:
fixed!!